### ISSUE TITLE
Ensure elements that have scrollable content are accessible by keyboard (.table-scroll)

### ISSUE THREAD

**Issue by mgifford** [mgifford/daily-dap#188]
Title: WCAG 2.1.1: Ensure elements that have scrollable content are accessible by keyboard (.table-scroll)
Tags: Accessibility, WCAG 2.1.1, scrollable-region-focusable

Issue: Ensure elements that have scrollable content are accessible by keyboard (scrollable-region-focusable - https://accessibilityinsights.io/info-examples/web/scrollable-region-focusable)

Target application: Daily DAP Report - 2026-04-18 - https://mgifford.github.io/daily-dap/docs/reports/daily/2026-04-18/index.html#content-density-heading

Element path: section[aria-labelledby="fpc-exclusion-heading"] > .table-scroll

Snippet: <div class="table-scroll">

How to fix: 
Fix any of the following:
  Element should have focusable content
  Element should be focusable

Environment: Microsoft Edge version 147.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
